features/course/people/remove_people.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Course people page
```

Comment:
@ORIGIN-Ticket-009
Feature: Remove People Page
As a logged instructor, on the course people page, I want to be able to remove students or teachers.